products/ckeditor.md

```diff
@@ -0,0 +1,35 @@
+---
+title: CKEditor
+category: framework
+permalink: /ckeditor
+releaseColumn: true
+activeSupportColumn: true
+extendedSupportColumn: true
+releasePolicyLink: https://ckeditor.com/docs/ckeditor5/latest/updating/versioning-policy.html
+iconSlug: ckeditor4
+
+releases:
+-   releaseCycle: "5"
+    eol: false
+    support: true
+    latest: "41.1.0"
+    link: https://github.com/ckeditor/ckeditor5/releases/tag/v41.1.0
+    latestReleaseDate: 2024-02-07
```

Comment:
Given the release cycle name (`5`) does not match the versions, latest versions updates can't be automated. I think it will be very hard to maintain over time, so maybe we shouldn't document latest versions at all ? WDYT ?

Reply:
I don't know much about how auto updates work, but if you think it will be hard to automate I trust you.
I removed the column.